tests/test_utilities.py

```diff
@@ -161,3 +162,27 @@ def test_patch_path(self, lookup_patch):
         assert lookup_patch.call_args[0][0] == 'main'
         assert os.path.isdir(lookup_patch.call_args[0][1])
         assert lookup_patch.call_args[1] == {}
+
+    def test_catalog_admin_url(self):
+        url = utils.get_catalog_admin_url_template()
+        assert "/admin/catalogs/catalog/{catalog_id}/change/" in url
+
+        with override_settings(COURSE_CATALOG_API_URL="localhost:18387/api/v1/"):
```

Comment:
Let's move these repetitive hostnames and URIs into variables instead of duplicating them.